hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/snapshot/SnapshotManager.java

```diff
@@ -96,7 +96,7 @@
   private final boolean snapshotDiffAllowSnapRootDescendant;
 
   private final AtomicInteger numSnapshots = new AtomicInteger();
-  private static final int SNAPSHOT_ID_BIT_WIDTH = 24;
+  private static final int SNAPSHOT_ID_BIT_WIDTH = 31;
```

Comment:
I believe @szetszwo recommended setting this to 28 bits for now (something less than 31).

Reply:
Sure @arp7  I will set to 28 then.